tools/depends/target/ffmpeg/Makefile

```diff
@@ -26,7 +26,10 @@ ifeq ($(CROSS_COMPILING), yes)
 endif
 ifeq ($(OS), linux)
   ffmpg_config += --target-os=$(OS) --cpu=$(CPU)
-  ffmpg_config += --enable-vdpau --enable-vaapi --enable-pic
+  ifneq ($(TARGET_PLATFORM),raspberry-pi)
+    ffmpg_config += --enable-vdpau
+  endif
+  ffmpg_config += --enable-vaapi --enable-pic
```

Comment:
in my opinion that "let's rebuild ffmpeg if we find a compatible one" thing is not compatible with depends. FindFFMPEG.cmake should fail and let distribution maintainers fix / update their ffmpeg instead of silently building it. but hey, it's just an opinion.